inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/trigger/PathPattern.java

```diff
@@ -168,8 +162,9 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object object) {
-        if (object instanceof PathPattern) {
-            PathPattern entity = (PathPattern) object;
+        if (object instanceof org.apache.inlong.agent.plugin.trigger.PathPattern) {
```

Comment:
I'm sorry but why replace PathPattern with org.apache.inlong.agent.plugin.trigger.PathPattern ?

Reply:
idea did it，，，